packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeDetailsItem/MT5TradeDetailsItem.tsx

```diff
@@ -29,7 +29,7 @@ const MT5TradeDetailsItem: FC<TMT5TradeDetailsItemProps> = ({ className, label,
                 <Text bold size='sm'>
                     {value}
                 </Text>
-                {variant === 'clipboard' && <Clipboard textCopy={value} />}
+                {variant === 'clipboard' && <Clipboard textCopy={value} tooltip='bottom' />}
```

Comment:
If the reason that you're using the tooltip is to work around the type error which we get, the fix was merged a few minutes ago

Reply:
the fix: https://github.com/binary-com/deriv-app/pull/12723